copier.yml

```
@@ -61,33 +57,14 @@ license:
     default: MIT
 
 project_slug:
-    when: "{{ template_mode == 'custom' }}"
     type: str
-    help: "What is the project slug? Used in hyperlinks."
+    help: "What is the project slug (all-lowercase name)? For example used to create an environment of that name."
     default: "{{ project_name|lower|replace(' ', '-') }}"
 
-dev_platform:
-    type: str
-    help: "Which development platform are you planning to use? Used in documentation links."
-    choices:
-        - "GitHub"
-        - "GitLab"
-    default: "GitHub"
-
-username:
-    when: "{{ use_git }}"
-    type: str
-    help: "Provide you or your organization's {{ dev_platform }} username. Used to generate certain documentation and hyperlinks."
-
-dev_platform_url:
-    type: str
-    when: false
-    default: "https://{{ dev_platform|lower }}.com"
-
 version:
     type: str
     default: 0.1.0
-    help: "Enter the version of the Python package"
+    help: "Enter the initial version of the project."
```

```suggestion
    help: "Enter the version of the project."
```
this would also be used to regenerate a project, where the version number would be lager.